plugins/woocommerce-admin/client/core-profiler/pages/BusinessInfo.tsx

```diff
@@ -347,9 +349,14 @@ export const BusinessInfo = ( {
 										geolocationOverruled || false,
 								},
 							} );
+							setHasSubmitted( true );
 						} }
 					>
-						{ __( 'Continue', 'woocommerce' ) }
+						{ hasSubmitted ? (
+							<Spinner />
```

Comment:
I wonder if an error occurs, will the state be reset?

Reply:
It won't, but it'll move to the next state if/when the API call promise rejects